test_doubly_linkedlist.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from doubly_linkedlist import Dll
+
+
+def test_dll():
+    dll = Dll([4, 6, 8, 2])
+
+    assert dll.head.value == 2
+    assert dll.head.next.value == 8
+    assert dll.head.next.next.previous.value == 8
+    assert dll._length == 4
+    assert dll.tail.value == 4
+
+
+def test_push():
+    dll = Dll([5, 7, 8])
+    dll.push(3)
+
+    assert dll.head.value == 3
+    assert dll._length == 4
+    assert dll.head.previous == None
+    assert dll.head.next.value == 8
+    assert dll.head.next.previous.value == 3
+    assert dll.tail.value == 5
+
+
+def test_push_empty():
+    dll = Dll()
+    dll.push("Head")
+
+    assert dll.head.value == "Head"
+    assert dll.tail.value == "Head"
+
+
+def test_append():
+    dll = Dll([])
+    dll.append(33)
+
+    assert dll.head.value == 33
+    assert dll.head.next == None
+    assert dll.tail.value == 33
+    assert dll._length == 1
+
+    dll.append(8)
+
+    assert dll.head.value == 33
+    assert dll.head.next.value == 8
+    assert dll._length == 2
+    assert dll.tail.value == 8
+    assert dll.tail.previous.value == 33
```

Comment:
.next

Reply:
isnt that covered in line 48? dll.head.next.value